test/membrane/blank_video_generator_test.exs

```diff
@@ -1,26 +1,41 @@
 defmodule Membrane.BlankVideoGeneratorTest do
-  use ExUnit.Case
+  use ExUnit.Case, async: true
   use Membrane.Pipeline
 
   import Membrane.Testing.Assertions
 
-  alias Membrane.{Buffer, BlankVideoGenerator}
+  alias Membrane.{BlankVideoGenerator, Buffer}
   alias Membrane.Caps.Video.Raw
+  alias Membrane.H264.FFmpeg.Encoder
   alias Membrane.Testing.{Pipeline, Sink}
 
-  test "Blank Video Generator should work with buffers as demand unit for all allowed caps" do
-    caps_i420 = %Raw{
-      format: :I420,
-      height: 720,
-      width: 1280,
-      framerate: {1, 1},
-      aligned: true
-    }
+  @caps_i420 %Raw{
+    format: :I420,
+    height: 720,
+    width: 1280,
+    framerate: {1, 1},
+    aligned: true
+  }
+  @caps_i422 %Raw{@caps_i420 | format: :I422}
 
-    caps_i422 = %Raw{caps_i420 | format: :I422}
+  describe "Blank Video Generator should work with buffers as demand unit for caps" do
+    test "with I420 format" do
+      test_for_caps(@caps_i420)
+    end
 
-    test_for_caps(caps_i420)
-    test_for_caps(caps_i422)
+    test "with I422 format" do
+      test_for_caps(@caps_i422)
+    end
+  end
+
+  describe "Generated raw video should be successfully encoded to H264 for caps" do
```

Comment:
Personally, I'm not a fan of ESpec-style, verbose descriptions - they obfuscate what the test is testing. In case of error here you're getting: `test Blank Video Generator should work with buffers as demand unit for caps with I420 format (Membrane.BlankVideoGeneratorTest)`.
I'd try to change it into `test I420 format buffer generation (Membrane.BlankVideoGeneratorTest)` as it contains all the needed info and is very concise - notice that I swapped the grouping order.
What's your take on that?